Editor/ProxyFeedbackEditor.cs

```diff
@@ -0,0 +1,143 @@
+
+            if (GUILayout.Button("Reload Data"))
+                Refresh();
+
+            if (GUILayout.Button("Clear Data"))
+            {
+                ClearData();
+                Refresh();
+            }
+
+            if (GUILayout.Button("Save Data"))
+            {
+                SaveData();
+                Refresh();
+            }
+
+            m_Scroll = GUILayout.BeginScrollView(m_Scroll);
+            var hasFeedback = false;
+            foreach (var kvp in m_SerializedFeedback)
+            {
+                GUILayout.Label(kvp.Key.Name, EditorStyles.boldLabel);
+                DrawPreferences(kvp.Value);
+                hasFeedback = true;
+            }
+
+            if (!hasFeedback)
+                GUILayout.Label("No serialized feedback");
+
+            GUILayout.EndScrollView();
+        }
+
+        static void DrawPreferences(SerializedProxyFeedback feedback)
+        {
+            GUILayout.Label("Left Hand");
+            DrawNode(feedback.leftNode);
+            GUILayout.Label("Right Hand");
+            DrawNode(feedback.rightNode);
+        }
+
+        static void DrawNode(SerializedProxyNodeFeedback node)
+        {
+            var keys = node.keys;
+            var values = node.values;
+            for (var i = 0; i < keys.Length; i++)
+            {
+                var data = values[i];
+                data.presentations = EditorGUILayout.IntField(keys[i].ToString(), data.presentations);
+            }
+        }
+
+        void Refresh()
+        {
+            m_SerializedFeedback.Clear();
+            var preferences = SerializedPreferencesModule.DeserializePreferences(Core.EditorVR.serializedPreferences);
+            if (preferences == null)
+                return;
+
+            foreach (var kvp in preferences.items)
+            {
+                var type = kvp.Key;
+                if (typeof(TwoHandedProxyBase).IsAssignableFrom(type))
+                {
+                    var item = kvp.Value;
+                    Type payloadType = null;
+                    foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
+                    {
+                        payloadType = assembly.GetType(item.payloadType);
+                        if (payloadType != null)
+                            break;
+                    }
+
+                    if (payloadType == null)
+                        continue;
+
+                    var payload = (SerializedProxyFeedback)JsonUtility.FromJson(item.payload, payloadType);
+                    m_SerializedFeedback[kvp.Key] = payload;
+                }
+            }
+        }
+
+        static void ClearData()
+        {
+            var preferences = SerializedPreferencesModule.DeserializePreferences(Core.EditorVR.serializedPreferences);
+            if (preferences == null)
+                return;
+
+            foreach (var kvp in new Dictionary<Type, SerializedPreferencesModule.SerializedPreferenceItem>(preferences.items))
```

Comment:
is it necessary to create a new dict ?  `preferences.items` is already a `Dictionary<Type, SerializedPreferenceItem>`

Reply:
We are removing elements